source/coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -154,3 +154,100 @@ Note that:
 - The `end` day is **inclusive**: it is the last day a variable and its formulas are active (and not the first day it is not active anymore).
 - The `end` value is a string of format `YYYY-MM-DD` where `YYYY`, `MM` and `DD` are respectively a year, month and day.
 - When defining a date, the month is given **before** the day.
+
+
+## Ending a parameter at a specific date
+
+Similarly to variables, parameters are prone to disappear as the legislation evolves.
+
+For instance, let's assume that the `housing_allowance` is only defined from the 1st of January 2010 to the 30th of November 2016.
```

Comment:
Is that a parameter or a variable?

Reply:
It's a[ parameter](https://github.com/openfisca/country-template/blob/master/openfisca_country_template/parameters/benefits/housing_allowance.yaml).

Reply:
I don't really want an answer here 😉 I was noting that this is not clear in the current phrasing and it should be made clear in the text 🙂